bin/www

```diff
@@ -4,22 +4,22 @@
  * Module dependencies.
  */
 
-var app = require('../app');
-var debug = require('debug')('flow:server');
-var http = require('http');
+const app = require('../app');
```

Comment:
Is there a way to run the linter on all our files cause it looks like it only fixes edited files? 😟

Reply:
Oh oops I missed those files for the linter, but yup I can fix that!

Reply:
Ooo nice!! :) Thank you!